bee-node/src/plugins/dashboard/mod.rs

```diff
@@ -161,11 +160,10 @@ where
                 users.clone(),
             );
 
-            info!("Dashboard available at http://localhost:{}.", config.port());
+            info!("Dashboard available at http://{}.", config.bind_socket_addr());
```

Comment:
I guess I would print that log only *after* the `warp` server has been spawned, because only then we know what IP the server was actually bound to. If you set `0.0.0.0` in your config, then this log output wouldn't be helpful as it would just replicate that.